bgpd/bgp_network.c

```diff
@@ -829,6 +829,14 @@ enum connect_result bgp_connect(struct peer_connection *connection)
 	if (CHECK_FLAG(peer->flags, PEER_FLAG_TCP_MSS))
 		sockopt_tcp_mss_set(connection->fd, peer->tcp_mss);
 
+	/* Neighbor's bind() as a source address when ip transparent is used */
+	if (CHECK_FLAG(peer->flags, PEER_FLAG_IP_TRANSPARENT) &&
+	    CHECK_FLAG(peer->flags, PEER_FLAG_UPDATE_SOURCE)) {
+		frr_with_privs(&bgpd_privs) {
+			sockopt_ip_transparent(connection->fd);
```

Comment:
Should we check the return code here, or we don't care? If we don't care then `sockopt_ip_transparent()` should be void?